homeassistant/components/whirlpool/__init__.py

```diff
@@ -20,7 +27,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Whirlpool Sixth Sense from a config entry."""
     hass.data.setdefault(DOMAIN, {})
 
-    auth = Auth(entry.data["username"], entry.data["password"])
+    backend_selector = BackendSelector(Brand.Whirlpool, Region.EU)
```

Comment:
Just out of curiosity, should the region and brand be configurable when setting up a config entry?

Reply:
I have that ready on a separate branch (config entry and such) and was planning to open the PR right after this one closes. I was trying to keep this PR short just to fix the integration. 

Before 0.17.0, the library only supported EU, so supporting US will be new.